common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -105,7 +106,7 @@
                 'entry': entry,
                 'body': capture(caller.body)
             }))
-        except IOError as e:
+        except (IOError, WebpackLoaderBadStatsError) as e:
             # Don't break Mako template rendering if the bundle or webpack-stats can't be found, but log it
             logger.error(e)
```

Comment:
Could you add more specific logging here, so we can easily get a splunk search for this error in production? Something as simple as prepending the JIRA ticket number would suffice.